src/app/dashboard/homebrew/items/pack-detail/pack-detail.component.css

```diff
@@ -2,6 +2,11 @@
   flex: 1 1 auto;
 }
 
+.item-list {
+  max-height: calc(100vh - 188px);
```

Comment:
Not sure what 188 and 236 are, but looks like it makes sense to variablize them.

Reply:
These are the height of the navbars+toolbars+footers - as far as I can tell there's no good way to get these from the rendered page (it looks like the EB client has these set as SASS variables, which I should probably switch to doing)